src/app.ts

```diff
@@ -1,9 +1,11 @@
 import { module } from 'angular';
 
+import { AMAZON_MODULE } from '@spinnaker/amazon';
 import { CORE_MODULE, ApplicationDataSourceRegistry } from '@spinnaker/core';
 import { KAYENTA_MODULE } from 'kayenta/canary.module';
 
 module('netflix.spinnaker', [
+  AMAZON_MODULE,
```

Comment:
Only affects Kayenta development. Needed for working on infrastructure provisioning stage.